com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -862,16 +868,17 @@ public struct TransformData : INetworkSerializeByMemcpy
             public NetworkObject OwnerObject;
             public ulong TargetClientId;
 
+            public int NetworkSceneHandle;
```

Comment:
why we have `NetworkSceneHandle` here and also in `HeaderData` above?

Reply:
Because the one in the header was a very early implementation and was not supposed to be there... >.<
Removing it.